docs/account/avatar.md

```diff
@@ -2,13 +2,12 @@
 
 #### How do I set an avatar for my account?
 
-1. Go to this site and login with your osu!droid account credentials if you haven’t logged in yet.
+1. Go to [this](https://osudroid.moe/user/?action=login) site and login with your osu!droid account credentials if you haven’t logged in yet.
 2. Open the left-side drawer and click on Profile.
-3. Press “Select file” and select an image file to upload as avatar. Keep in mind that there is a file size limitation of 1 MB.
+3. Press “Select file” and select an image file to upload as avatar. Keep in mind that there is a file size limitation of around 1.5 MB.
 4. Crop your image using the tools provided if necessary.
 5. Save your image by clicking on the blue checkmark button on the bottom-right side of your screen.
```

Comment:
```suggestion
5. Save your image by clicking on the blue checkmark button on the bottom-right side of your screen (You won't see any feedback upon clicking it, don't worry about it).
```
maybe? the fact that the site won't tell user that they saved their cropped image can be confusing to them

Reply:
Works for me.